core/src/main/java/org/geysermc/geyser/entity/type/living/ArmorStandEntity.java

```diff
@@ -336,17 +353,10 @@ private void updateSecondEntityStatus(boolean sendMetadata) {
             secondEntity.getDirtyMetadata().put(EntityData.BOUNDING_BOX_WIDTH, 0.0f);
             secondEntity.getDirtyMetadata().put(EntityData.BOUNDING_BOX_HEIGHT, 0.0f);
             secondEntity.spawnEntity();
```

Comment:
Now that we don't return when the entity is null, I think this could try to spawn the entity when it's already spawned.